labelbox/data/annotation_types/geometry/rectangle.py

```diff
@@ -30,6 +32,28 @@ def geometry(self) -> geojson.geometry.Geometry:
             [self.start.x, self.start.y],
         ]])
 
+    @classmethod
+    def from_shapely(cls, shapely_obj: SPolygon) -> Rectangle:
```

Comment:
Do these type hints work? I thought you need to put "Rectangle" in quotes because technically Rectangle isn't defined until you are outside the scope of these function definitions (Same with all the other types)

Reply:
They work if you include `from __future__ import annotations` at the top of the file

Reply:
https://www.python.org/dev/peps/pep-0563/#enabling-the-future-behavior-in-python-3-7

Reply:
Ahh thinking about this more, this is only for Python 3.7 and we are still doing Python 3.6-3.8.

I will go with the quote method for now but once we go to Python 3.7-3.9 I think we can do `from __future__ import annotations` if we want to